contexts-config-store/store/contexts_config_store_impl.go

```diff
@@ -93,6 +93,11 @@ func (store *contextConfigStoreImpl) AddNewContext(newContext *generated.Kurtosi
 	store.Lock()
 	defer store.Unlock()
 
+	if newContext.GetName() == persistence.DefaultContextName {
+		return stacktrace.NewError("Adding a new context names '%s' is not allowed as it is the reserved context "+
```

Comment:
s/names/with name/g
 s/it is the/it is a/g

We say that contexts have equality based on uuid but here we are using name as proxy. I think its alright!